template.yaml

```diff
@@ -15,7 +15,7 @@ Globals:
       AllowOrigin: !Sub
```

Comment:
Simplify Cors definition without using parameters and secrets manager.

  Api:
    Cors:
      AllowHeaders: "'Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token'"
      AllowMethods: "'OPTIONS, GET, PUT, POST'"
      AllowOrigin: "'*'"